athletes/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Trainer (models.Model):
+    first_name = models.CharField(max_length=40, null=False, blank=False)
+    last_name = models.CharField(max_length=40, null=True, blank=False)
+
+
+class Athlete (models.Model):
+    first_name = models.CharField(max_length=40,null=False, blank=False)
+    last_name = models.CharField(max_length=40, null=True, blank=False)
+    height = models.FloatField()
+    birth_date = models.DateField(null=True, blank=True)
+    birth_place = models.CharField(max_length=40, null=True, blank=False)
+    picture_url = models.ImageField(upload_to='media', null=True)
```

Comment:
La resolucion de la ruta de imagenes no esta funcionando. Podrias usar el utilitario del proyecto de ejemplo 
[https://github.com/larry852/krono-backend/blob/master/core/utils/file.py](url)
```suggestion
    avatar = models.ImageField(upload_to=get_path_class, default='default-profile.png')
```
Y dejarle un default para que cuando pasemos al front no muera. 

la default esta en [https://github.com/larry852/krono-backend/blob/master/media/default-profile.png](url)
seria dejarlo en el media de nuestro proyecto

Reply:
Tambien agrega `media/*/` a las excpeciones del gitignore asi no subimos images que probemos en local al repo

Reply:
Listo

Reply:
Tambien es necesario tener esta definicion de la ruta de media sobre routers
`urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)`
algo asi [https://github.com/larry852/krono-backend/blob/master/core/urls.py](url)